src/service/poc_mobile.proto

```diff
@@ -4,6 +4,21 @@ package helium.poc_mobile;
 
 import "mapper.proto";
 
+message subscriber_event_req_v1 {
+  // Pub key for the subscriber's NFT
```

Comment:
_helium_ pub key ideally.. I think we should try to keep all keys in oracles as close to helium keys until we sweep them all over to Solana